src/Microsoft.AspNet.JsonPatch/Helpers/PathHelpers.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.JsonPatch.Helpers
+{
+    internal static class PathHelpers
+    {
+        internal static CheckPathResult CheckPath(string pathToCheck)
```

Comment:
Was used to allow logging in the calling class (return CheckPathResult to JsonPatchDoc, where LogErrorAction is) - but as logging shouldn't be done there, this can indeed be changes to PathHelpers.NormalizePath